nodes/src/nodes/telegram/IEndpoint.py

```
@@ -0,0 +1,425 @@
+                poll_count += 1
+                ok = data.get('ok')
+                updates = data.get('result', []) if ok else []
+
+                if ok:
+                    for update in updates:
+                        uid = update.get('update_id')
+                        await self._handle_update(update)
+                        offset = uid + 1
+
+            except asyncio.CancelledError:
+                break
```

Ruff (0.15.10)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)